_posts/2023-03-29-langchain-slackbot.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+author: "Alex Strick van Linschoten"
+title: "TITLE TK"
+description: "DESCRIPTION TK"
+category: zenml
+tags: tooling zenml evergreen zenml-project nlp llm
+publish_date: March 30, 2023
+date: 2023-03-30T00:02:00Z
+thumbnail: /assets/posts/slackbot/slackbot-small.png
+image:
+  path: /assets/posts/slackbot/slackbot.png
+---
+
+**Last updated:** March 30, 2023
+
+![*Image generated by [Midjourney v5](https://www.midjourney.com/)*](/assets/posts/slackbot/slackbot-small.png)
+
+The ZenML Slack bot is [live](https://zenml.io/slack-invite/)! 🥳
+
+We noticed the excitement around large language models and wanted to see how some of the new, developing paradigms fit in with ZenML and how MLOps workflows and standards usually work. So we built and deployed a Slack bot interface to help with community support. (Feel free to `@zenml-bot` over in our Slack to get help with your queries about how to use ZenML.)
```

Comment:
Id start stronger than this.. Something like "While Large Language Models are all the hype these days, there isnt enough talk about how to produtionalize them in an actual real world setting. This is why we decided to integrate a LLM into a production-ready workflow ..."